README.md

```diff
@@ -210,6 +210,36 @@ Valid patch directories include:
 
 **Nota Bene:** You need to be running at least Bash v4 to use this command.
 
+### `e get`
```

Comment:
To avoid confusion with [`electron/get`](https://github.com/electron/get) we should probably name this something else

Reply:
Renamed to `e show` in 307f949